Worksheet B/Pseudocode.MD

```diff
@@ -0,0 +1,38 @@
+```
+
+
+Word Bank: Repulsive, Bubble, Men, Roasted, Sophisticated,
```

Comment:
The words should all be the same length, otherwise the similarity score doesn't make sense